portal/src/texts/profile/animation.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Animasjon
+path: /profil/animasjon
+order: 8
+---
+
+# Animasjon
+
+<Ingress>
+    <p>Animasjon er et effektivt virkemiddel for å veilede brukerne og styre fokus mot det som er viktig. Det kan skape sømløse overganger og gi vår visuelle profil et moderne uttrykk.</p>
+</Ingress>
+
+## Hvorfor?
+Vi bruker animasjon for å:
+* Skape særpreg
+* Gi en bedre brukeropplevelse
+* Hjelpe og rettlede brukeren
+* Skape fokus
+* Kamuflere nedlastingstid (underholde)
+
+Husk at animasjon alltid skal kunne slås av dersom en bruker ikke ønsker dette.
```

Comment:
Viktig: Animasjon skal kunne slås av dersom en bruker ikke ønsker dette. Sørg for at løsningen fungerer som den skal og fortsatt er logisk uten dette ekstra virkemiddelet.